lib/simple_enum.rb

```diff
@@ -12,7 +12,11 @@
 
 require 'simple_enum/version'
 require 'simple_enum/attribute'
-require 'simple_enum/translation'
+begin
+  require 'simple_enum/translation'
+rescue SyntaxError
```

Comment:
probably should also be ArgumentError here. On Ruby 2.6 i get ArgumentError rather than SyntaxError.